provprofile/provprofile.go

```diff
@@ -106,6 +106,40 @@ func CreateProvisioningProfileModelFromFile(filePth string) (ProvisioningProfile
 	return provProfileData, nil
 }
 
+func walkProvProfiles(ppWalkFn func(provProfile ProvisioningProfileFileInfoModel) error) error {
+	absProvProfileDirPath, err := pathutil.AbsPath(provProfileSystemDirPath)
+	if err != nil {
+		return errors.Wrap(err, "Failed to get Absolute path of Provisioning Profiles dir")
+	}
+
+	pths, err := filepath.Glob(absProvProfileDirPath + "/*.mobileprovision")
```

Comment:
i would recommand: `filepath.Join(absProvProfileDirPath, "*.mobileprovision")` instead of: `absProvProfileDirPath + "/*.mobileprovision"`